docs/release-process.md

```diff
@@ -26,14 +48,47 @@ It's not (yet) quite an ideal fit for our workflow, but it helps! Steps:
       * `git merge --ff-only origin/release-v{MAJOR}.{MINOR}.x` # to pull in the latest on the branch
       * `git merge origin/main` # to pull in the changes you want to release
 
-1. Run `cargo release {MAJOR}.{MINOR}.{PATCH}` to perform a dry-run and check that the things
-   it is proposing to do seem reasonable. (XXX - note that the output here isn't actually
-   helpful - the `cargo` output will reflect the old previous version because it will be
-   building without having touched the version numbers in the `Cargo.toml`, and it doesn't
-   actually propose that it's going to do anything! But it's probably worthwhile anyway!)
-1. Run `cargo release -x {MAJOR}.{MINOR}.{PATCH}` to perform real run and to
-   bump version numbers and *publish the release to crates.io* and *create the tag*.
-   It does not push the tag or branch to github, but it will publish to crates.io, so
-   take care!
-1. Push your branch, and make a PR to request it be merged to the main branch.
-    * `git push origin --tags`
+1. Test the release using a dry-run:
+   * `cargo release-backend-crates {MAJOR}.{MINOR}.{PATCH}` to test the `uniffi_*` crates
+   * `cargo release-uniffi {MAJOR}.{MINOR}.{PATCH}` to test the `uniffi` crates
+   * Note: The MAJOR/MINOR/PATCH numbers will be different for the two runs
+   * Note: some of the output here isn't actually helpful - the `cargo` output will reflect the old
+     previous version because it will be building without having touched the version numbers in the
+     `Cargo.toml`, and it doesn't actually propose that it's going to do anything! But it's probably
+     worthwhile anyway!.
+
+1. Perform a real run to pump the version numbers, *publish the release to crates.io* and *create
+   the git tag*.
+   * Take care because this will publish the new releases on crates.io.
+   * This will create a local git tag, but does not push it to github.
+   * `cargo release-backend-crates -x {MAJOR}.{MINOR}.{PATCH}` to release the `uniffi_*` crates
+   * `cargo release-uniffi -x {MAJOR}.{MINOR}.{PATCH}` to release the `uniffi` crates
+
+1. Push your branch: `git push origin --tags`
```

Comment:
Is there a step missing here to update `uniffi/Cargo.toml` with the new version numbers of the sub-crates - there's also a bit of a chicken-and-egg situation here - don't the sub-crates need to be on crates.io before we can release the top-level crate?

Reply:
I think that `cargo release-backend-crates` should do that, although I'm not sure.  I'll verify that when I do the 0.23 release.  I added a warning to that section.